python/tests/res/job_queue/test_jobmanager.py

```diff
@@ -376,6 +377,15 @@ def test_data_from_forward_model_json(self):
             self.assertNotIn( "DATA_ROOT" , os.environ )
             self.assertNotIn( "ERT_RUN_ID", os.environ )
 
+    def test_complete(self):
+        with TestAreaContext("json_from_forward_model_NO_DATA_ROOT"):
+            with open("jobs.json", "w") as f:
+                f.write(JSON_STRING_NO_DATA_ROOT)
+
+            jobm = JobManager()
+            jobm.complete()
+            st = ForwardModelStatus.load(os.getcwd())
+            self.assertTrue( isinstance(st.end_time, datetime.datetime))
```

Comment:
Looks good! Could you add a statement checking that `start_time` is also set? Perhaps also checking that they are close to `now`?

Reply:
OK - done.